tensorflow/docs_src/get_started/get_started_for_beginners.md

```diff
@@ -611,7 +611,7 @@ def eval_input_fn(features, labels=None, batch_size=None):
         # No labels, use only features.
         inputs = features
     else:
-        inputs = (features, labels)
+        inputs = (dict(features), labels)
```

Comment:
Thanks for the fix, but I think you would expect to pass the features as a dict to your eval_input_fn (conversion from panda dataframe to dict should happen sometime before that)